### ISSUE TITLE
s3select:test_progress_expressions wrong key

### ISSUE THREAD

**Issue by alphaprinz** [ceph/s3-tests#508]
Hello,

In s3select:test_progress_expressions, the response is parsed with key 'Details'.
Eg
s3select_assert_result(obj_size, res_s3select_response[total_response-3]['Details']['BytesScanned'])
However the spec for Stats message (https://docs.aws.amazon.com/AmazonS3/latest/API/RESTSelectObjectAppendix.html under Stats Message, Payload specification) specifies 'Stats' field in the response xml.

In short, the Details field should be changed to Stats.
